Source/Core/DolphinQt/CheatSearchWidget.cpp

```diff
@@ -61,13 +63,25 @@ CheatSearchWidget::CheatSearchWidget(std::unique_ptr<Cheats::CheatSearchSessionB
   UpdateGuiTable();
 }
 
-CheatSearchWidget::~CheatSearchWidget() = default;
+CheatSearchWidget::~CheatSearchWidget()
+{
+  auto& settings = Settings::GetQSettings();
+  settings.setValue(QStringLiteral("cheatsearchwidget/displayhex"),
+                    m_display_values_in_hex_checkbox->isChecked());
+  if (m_session->IsIntegerType())
+  {
+    settings.setValue(QStringLiteral("cheatsearchwidget/parsehex"),
+                      m_parse_values_as_hex_checkbox->isChecked());
+  }
+}
 
 Q_DECLARE_METATYPE(Cheats::CompareType);
 Q_DECLARE_METATYPE(Cheats::FilterType);
 
 void CheatSearchWidget::CreateWidgets()
 {
+  auto& settings = Settings::GetQSettings();
```

Comment:
Is there a reason this is up at the top?  Could you move it down to closer where it is used?